ci/test/00_setup_env_native_qt5.sh

```diff
@@ -17,4 +17,4 @@ export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export PREVIOUS_RELEASES_TO_DOWNLOAD="v0.15.2 v0.16.3 v0.17.2 v0.18.1 v0.19.1"
 export BITCOIN_CONFIG="--enable-zmq --with-libs=no --with-gui=qt5 --enable-glibc-back-compat --enable-reduce-exports
---enable-debug --disable-fuzz-binary  CFLAGS=\"-g0 -O2 -funsigned-char\" CXXFLAGS=\"-g0 -O2 -funsigned-char\" --with-boost-process"
+--enable-debug CFLAGS=\"-g0 -O2 -funsigned-char\" CXXFLAGS=\"-g0 -O2 -funsigned-char\" --with-boost-process"
```

Comment:
This fails, because this pull only fixes the *windows* build, not the gcc build

Reply:
I couldn't follow. Which gcc bug do you mean?

Reply:
```
/usr/bin/ccache g++ -m64 -std=c++17 -DHAVE_CONFIG_H -I. -I../src/config  -DDEBUG -DDEBUG_LOCKORDER -DABORT_ON_FAILED_ASSUME  -I. -I./secp256k1/include  -DBOOST_SP_USE_STD_ATOMIC -DBOOST_AC_USE_STD_ATOMIC -pthread -isystem /tmp/cirrus-ci-build/depends/x86_64-pc-linux-gnu/include -I./leveldb/include -I./leveldb/helpers/memenv -I./univalue/include -D_GLIBCXX_DEBUG -D_GLIBCXX_DEBUG_PEDANTIC -I/tmp/cirrus-ci-build/depends/x86_64-pc-linux-gnu/include/  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS -DPROVIDE_MAIN_FUNCTION -O0 -g3 -ftrapv -fstack-reuse=none -Wstack-protector -fstack-protector-all   -Werror=vla -Werror=switch -Werror=unused-variable -Werror=date-time -Werror=return-type -Werror=sign-compare   -fPIE -pipe -O1 -g0 -O2 -funsigned-char -fno-extended-identifiers -c -o test/fuzz/test_fuzz_fuzz-cuckoocache.o `test -f 'test/fuzz/cuckoocache.cpp' || echo './'`test/fuzz/cuckoocache.cpp
In file included from test/fuzz/cuckoocache.cpp:5:0:
./cuckoocache.h: In instantiation of ‘void CuckooCache::cache<Element, Hash>::insert(Element) [with Element = bool; Hash = {anonymous}::RandomHasher]’:
test/fuzz/cuckoocache.cpp:42:67:   required from here
./cuckoocache.h:431:22: error: no matching function for call to ‘swap(std::__debug::vector<bool>::reference, bool&)’
             std::swap(table[last_loc], e);
             ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
```

https://cirrus-ci.com/task/6298763082858496?command=ci#L3984

Reply:
Yes, I see the error message. It seems provoked by depends with `DEBUG=1`.

Reply:
Another example of broken Windows builds with `DEBUG=1` is #19772.